src/ai/backend/common/validators.py

```diff
@@ -711,6 +711,19 @@ def check_and_return(self, value: Any) -> set:
             self._failure("value must be Iterable")
 
 
+class ToList(t.List):
```

Comment:
Could we just reuse [`trafaret.base.List`](https://github.com/Deepwalker/trafaret/blob/4ac577155c93e5a3b024645abd64b1442fd4d550/trafaret/base.py#L876)?

Reply:
I want to allow any iterable data type here since sequential data fetched from RPC response get deserialized into `tuple`.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/qqKjbYtIoo1cw3D1CMWW/f8dee5ec-7e6b-4717-979e-c93767430d07.png)